### ISSUE TITLE
View count for new stories updates infrequently on title page.

### ISSUE THREAD

**Issue by TrickQuestion** [knighty/fimfiction-issues#170]
So, on the mane page there are like lots of stories in the New and Popular Stories sequences marked "0 views", including one I just submitted. This is clearly false (otherwise, why would they end up in the Popular Stories box?) and I'm a little confused why this occurs. For one, there's a story published _after_ mine with 4 views and mine is still 0 views.

Is the view count being reduced by X views, or something...?

**EDIT:** This appears to have something to do with the system updating view counts sporadically, though when the updates occur is unpredictable. The only time this seems to be an issue (to me) is when a story is relatively new, but that's what the New and Featured sections are intended to do.

**Comment by ocalhoun6** [comment]
Yeah ... I know that tracking views puts a load on the server, but maybe it could be tweaked to do it a _little_ more often, especially on newly posted stories and/or stories that are currently 'hot'?

It's a good way to gauge how well a story is being received early on, but the sporadic updates make the number almost meaningless in the short term.
